sdk/deviceupdate/azure-iot-deviceupdate/pom.xml

```diff
@@ -29,6 +29,10 @@
   <properties>
     <jacoco.min.linecoverage>0.25</jacoco.min.linecoverage>
```

Comment:
Good to remove?

Reply:
Do you mean remove the whole pom.xml file?

Reply:
no the <jacoco.min.linecoverage> that is to cover for the testing gaps. I believe now that you have added sync tests it should be good to remove these.